action.yml

```diff
@@ -9,4 +9,4 @@ runs:
   # this is built using GCB by building the Dockerfile in this directory on
   # every tagged release.  After tagging a release, a new version should
   # automatically appear.
-  image: 'docker://gcr.io/kubebuilder/pr-verifier:v0.4.1'
```

Comment:
I think this should be v0.4.3 now.

We wouldn't want a kube-builder v0.4.3 relesae that points to a v0.4.2 image?

Reply:
I'm happy to make the change to v0.4.3 and have done so since it's easy enough to revert - adding @killianmuldoon 's point below this PR should probably only be merged if a 0.4.3 release if cut right after, otherwise the v0.4.3 image won't actually exist.